### ISSUE TITLE
`scoreboard()` function returning an error

### ISSUE THREAD

**Issue by rbarton65** [rbarton65/espnff#11]
```python3
>>> league.scoreboard()
File "/usr/lib/python3.5/site-packages/espnff-1.2.2-py3.5.egg/espnff/espnff.py", line 120, in scoreboard
  File "/usr/lib/python3.5/site-packages/espnff-1.2.2-py3.5.egg/espnff/espnff.py", line 120, in <listcomp>
  File "/usr/lib/python3.5/site-packages/espnff-1.2.2-py3.5.egg/espnff/espnff.py", line 178, in __init__
  File "/usr/lib/python3.5/site-packages/espnff-1.2.2-py3.5.egg/espnff/espnff.py", line 188, in _fetch_matchup_info
IndexError: list index out of range
```

**Comment by rbarton65** [comment]
At this moment the `Matchup` class isn't accounting for bye weeks. Will have this fixed in a patch.

**Comment by rbarton65** [comment]
this issue is fixed in patch `v1.2.3`
